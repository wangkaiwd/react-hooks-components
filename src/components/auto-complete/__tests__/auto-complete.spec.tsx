import renderer from "react-test-renderer";
import AutoComplete from "../auto-complete";

describe("AutoComplete:", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<AutoComplete dataSource={[{ id: 1, label: "1" }]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("should display list when input focus", () => {

  });
  it("should hide list when input blur", () => {

  });
  it("should filter content by input text", () => {

  });
});
