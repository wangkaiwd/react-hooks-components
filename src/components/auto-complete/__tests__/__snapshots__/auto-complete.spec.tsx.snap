// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoComplete: should render correctly 1`] = `
<div
  className="ant-auto-complete"
>
  <div
    className="ant-input"
  >
    <input
      data-testid="input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="Please input something..."
    />
  </div>
</div>
`;
